operator/gitops/argocd/pipeline-service/openshift-pipelines/pipelines-scc.yaml

```diff
@@ -0,0 +1,37 @@
+---
+kind: SecurityContextConstraints
+metadata:
+  name: pipelines-scc
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
```

Comment:
I mentioned this in @ramessesii2 PR on this as well .... some comments on why we need the deltas from the restricted SCC would help here

Reply:
I suspect they are related to building with buildah ... did you perhaps just copy what was being done from the pipeline SA, or is this a delta of what was done for the pipeline SA ?

Reply:
Those permissions were provided by @Michkov and ensures that build-service can run its pipelines.
We are temporarily responsible for those permissions, and we expect those changes to be reverted (which is why they are bundled in their own commit).

Reply:
I also tried comparing the one created automatically by Pipelines to the one provided by Michal and they seem the same to me. It's just we'll be creating them instead on Pipelines for now.

Reply:
ok so yes to my question (buildah related == what Michal would provide) and they are temporary